Apps/W1/Shopify/app/src/Customer/Codeunits/ShpfyCustomerImport.Codeunit.al

```diff
@@ -49,12 +47,12 @@ codeunit 30117 "Shpfy Customer Import"
     /// Get Customer.
     /// </summary>
     /// <param name="ShopifyCustomerResult">Parameter of type Record "Shopify Customer".</param>
-    internal procedure GetCustomer(var ShopifyCustomerResult: Record "Shpfy Customer")
+    procedure GetCustomer(var ShopifyCustomerResult: Record "Shpfy Customer")
     begin
         ShopifyCustomerResult := ShopifyCustomer;
     end;
 
-    internal procedure SetAllowCreate(Value: Boolean)
+    procedure SetAllowCreate(Value: Boolean)
```

Comment:
we should have a good documentation for all procedures that aren't marked as internal.